engine/source/materials/mat/mat076/asso_plas76.F

```diff
@@ -65,7 +64,7 @@ SUBROUTINE ASSO_PLAS76(
      .   SIGOXX(NEL),SIGOYY(NEL),SIGOZZ(NEL),
      .   SIGOXY(NEL),SIGOYZ(NEL),SIGOZX(NEL),
      .   EPSD(NEL),DPLANL(NEL)
-      TYPE (TTABLE), DIMENSION(NUMTABL) ,TARGET  ::  TABLE
+      TYPE (TABLE_4D_), DIMENSION(NUMTABL) ,TARGET  ::  TABLE
```

Comment:
Illegal `TARGET` attribute, because TABLE is an argument, and there is no explicit interface.

Since the TARGET attribute is not really needed, I would suggest to remove it, and to use `TABLE(ID_FUNC_TRAC)` , with `INTEGER, PARAMETER :: ID_FUNC_TRAC =1` instead of `FUNC_TRAC`